Header header
int8 done
string group_name
int64 capable_candidate_count
moveit_msgs/RobotState[] capable_candidates
float32[] candidate_costs
int64 neighbor_count
float32[3] neighbor_position_average
float32[3] pointing_vector_variance
float32[3] facing_vector_variance
float32 pointing_dot_product_average
float32 facing_dot_product_average

